agent/agent_endpoint.go

```diff
@@ -1444,7 +1444,7 @@ func (s *HTTPHandlers) AgentToken(resp http.ResponseWriter, req *http.Request) (
 				triggerAntiEntropySync = true
 			}
 
-		case "acl_agent_master_token", "agent_master":
+		case "acl_agent_master_token", "agent_master", "agent_recovery":
```

Comment:
Is the plan to deprecate the `acl_agent_master_token` and `agent_master` naming in favour of `agent_recovery`? If so I don't see any deprecation notice for it in the doc.
It also would be nice to have those defined as const and have a deprecation notice for the ones we want to remove in the future. 
What you think about that?

Reply:
Yeah. It's not super obvious from the diff, but the `target` we're switching on is a path suffix - so there's a deprecation notice in the API docs [here](https://github.com/hashicorp/consul/blob/44d8abd341ca33c5b927aa473562a61ee8bcfd01/website/content/api-docs/agent/index.mdx?plain=1#L742-L758) 🙂 

I'd be happy to move them into constants, but as they're only really used in this handler, I wonder if the indirection would be worth it, or make the code less clear?